api/controllers/UserController.js

```diff
@@ -287,26 +310,49 @@ module.exports = {
         delete request.payload.tester;
       }
 
+      // Not showing request payload to avoid showing user passwords in logs.
       const user = await User.create(request.payload);
       if (!user) {
+        // Not showing request payload to avoid showing user passwords in logs.
```

Comment:
As an alternative, you can always do some string replacement on the sensitive property. See this line in Snap Service where we want to provide debug info, but also want to avoid leaking sensitive data into logs:

https://github.com/UN-OCHA/tools-snap-service/blob/e68e6ffda5cb0d607f290cba4d7a45b4ff175e16/app/app.js#L251